content/plans.ts

```diff
@@ -52,7 +52,7 @@ export const Plans: PlanProps[] = [
 		billsResources: true,
 		features: [
 			{
-				feature: 'Up to 15 projects',
+				feature: 'Up to 3 projects',
```

Comment:
@oddgrd Juan mentioned let's keep this as 'More projects' ambiguously, and mention in the launch announcement that it is subject to change